core/templates/pages/exploration-player-page/learner-experience/conversation-skin.directive.html

```diff
@@ -276,6 +276,9 @@
     z-index: 3;
   }
 
+  .conversation-skin-supplemental-card-container {
+    margin-bottom: 200px;
```

Comment:
Can we add this to the existing block on line 298?

Reply:
But that block is under the media query for width<=360px but we need this margin for width greater than that.

Reply:
I think it's not inside the media query block, can you please check again? :)

Reply:
ok. Sorry I didn't check that before. I thought that after that all the media-query blocks are there so I wrote before first media query block.
Now it worked and I pushed the code. PTAL Thanks.
@DubeySandeep

Reply:
@DubeySandeep I think there is the issue that lint tests timeout instead of producing an error for this issue.

Reply:
@vojtechjelinek, The issue is resolved in #11672 :)

Reply:
Great, thanks a lot.